lib/signaling/v2/cancelableroomsignalingpromise.js

```diff
@@ -22,12 +22,12 @@ function createCancelableRoomSignalingPromise(token, ua, localParticipant, iceSe
 
   var peerConnectionManager = new PeerConnectionManager(iceServerSource, encodingParameters, preferredCodecs, options);
 
-  var mediaStreamTracksAndDataTrackSenders = flatMap(localParticipant.tracks, function(track) {
-    return [track.mediaStreamTrackOrDataTrackTransceiver];
+  var trackSenders = flatMap(localParticipant.tracks, function(trackV2) {
+    return trackV2.trackTransceiver;
```

Comment:
If we `flatMap`, we should return an Array here, even if it's trivial, right? (I believe the implementation is using `concat`, which will accept singular values, but imagine if we were typing this with TypeScript—I believe this would fail to type.)